### DISCUSSION TITLE
common_threads.h header dosyası

### DISCUSSION THREAD

**Discussion by AybukeNr** [gusanmaz/BMB311_OS_2023#8] [Q&A] [answered]
Hocam 27.2 de kodlara common_threads.h header dosyası kodlara ekli fakat kodlar bu header olmadan da çalışıyor, pthread_create işlevi için zaten pthread.h'ı kullanmış ve çıktıda bir sorun olmuyor, videoda bu headera yer vermememiz sorun olur mu

**Comment by gusanmaz** [comment, marked as answer]
"common_threads.h" icinde hata kontrolu amacli wrapper fonksiyonlari tanimlaniyor. 

- https://github.com/remzi-arpacidusseau/ostep-code/tree/master/include baglantisindaki dosyalari indirip `common_threads.h` ve `common.h` dosyalarini calistiracaginiz koda `#include` ifadesi ile ekleyerek kullanabilirsiniz. 
- Ya da tirnak isareti ile sarmallanmis `#include` ifadelerini kitapta verilen ornek kodlardan silip; kodlardaki buyuk harf ile baslayan fonksiyon cagrilarinin ilk harfini kucuk harf yaparak ilgili kodlari calistirabilirsiniz.
- Hangi yolu tercih ederseniz edin https://github.com/remzi-arpacidusseau/ostep-code/tree/master/include baglantisindaki dosyalari incelemenizi tavsiye ederim.
